src/TaskManager/API/ExecutionStatus.cs

```diff
@@ -23,7 +23,7 @@ public class ExecutionStatus
         /// <summary>
         /// Gets or sets the status of the execution.
         /// </summary>
-        public TaskExecutionStatus Status { get; set; } = TaskExecutionStatus.Unknown;
+        public TaskExecutionStatus Status { get; set; }
```

Comment:
shouldn't we be instantiating it with data otherwise its type should be "TaskExecutionStatus?" which is a headache?

Reply:
is intention to remove TaskExecutionStatus.Unknown completely? should it be removed from the enum itself?

Reply:
Can always instantiate it with created yeah